src/app/api/main.py

```
@@ -116,12 +130,127 @@ async def lifespan(app: FastAPI):
+):
+    statement = select(JobMetadata).where(
+        JobMetadata.user_id == session_from_db.user_id,
+        JobMetadata.session_id == thread_id,
+    )
+    results = session.exec(statement).all()
+    return results
+
+@app.post("/jobs/{job_id}/toggle-pause", status_code=200)
+async def toggle_pause(
+    job_id: str,
+    session: Session = Depends(get_sqlmodel_session),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)